main.js

```diff
@@ -15,13 +19,132 @@ navLink.forEach((n) => n.addEventListener('click', () => {
   navMenu.classList.remove('active');
 }));
 
-formSubmit.addEventListener('submit', (e) => {
-  const emailFormat = /^[a-z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-z0-9-]+(?:\.[a-z0-9-]+)*$/;
+function getFormInput() {
+  const data = {
+    name: userName.value,
+    email: email.value,
+    message: message.value,
+  };
+  localStorage.setItem('FormInput', JSON.stringify(data));
+}
+
+form.addEventListener('input', () => {
+  getFormInput();
+});
 
+form.addEventListener('submit', (e) => {
+  const emailFormat = /^[a-z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-z0-9-]+(?:\.[a-z0-9-]+)*$/;
   if (email.value.match(emailFormat)) {
     errorMessage.innerHTML = '';
   } else {
     e.preventDefault();
-    errorMessage.innerHTML = 'Email is invalid, must be in this format example@xyz.com';
+    errorMessage.innerHTML = 'Email is incorrect, field should not be empty. <br/> should be in this format example@xyz.com';
+  }
+});
+
+const projectInfo = [
```

Comment:
- [ ] **[OPTIONAL]** There appears to be an issue with the responsiveness of your modal. It would be nice if you fixed this issue. From about **682px** of screen width and upwards the design of the mobile version of your modal leaves a lot of blank page spaces and looks somewhat different from the Figma template. See image below:

![ricky-res-res](https://user-images.githubusercontent.com/36542986/136659290-45649592-650c-422b-95be-87399cfdd686.jpg)

Reply:
Sure I'll go ahead and fix it. Thank you very much @TedLivist